src/eva/vision/data/datasets/segmentation/consep.py

```diff
@@ -145,23 +142,11 @@ def _load_file_paths(self, split: Literal["train", "val"] | None = None) -> List
 
         return sorted(paths)
 
-    def _get_coords(self, index: int) -> Tuple[Tuple[int, int], int, int]:
-        """Returns the coordinates ((x,y),width,height) of the patch at the given index."""
-        image_index = self._get_dataset_idx(index)
-        patch_index = index if image_index == 0 else index - self.cumulative_sizes[image_index - 1]
-        coords = self.datasets[image_index]._coords
-        return coords.x_y[patch_index], coords.width, coords.height
-
     def _get_mask_path(self, index: int) -> str:
         """Returns the path to the mask file corresponding to the patch at the given index."""
         filename = self.filename(index).split(".")[0]
-        mask_dir = "Train" if filename.startswith("train") else "Test"
-        return os.path.join(self._root, mask_dir, "Labels", f"{filename}.mat")
-
-    def _extract_mask_patch(self, index: int, mask: npt.NDArray[Any]) -> npt.NDArray[Any]:
-        """Reads the mask patch at the coordinates corresponding to the given patch index."""
-        (x, y), width, height = self._get_coords(index)
-        return mask[x : x + width, y : y + height]
+        mask_dir = "Train/Labels" if filename.startswith("train") else "Test/Labels"
+        return os.path.join(self._root, mask_dir, f"{filename}.mat")
```

Comment:
I think we shouldn't change that,  because windows used for example will have issues, as we have the `/` pre-determined here for the split

so this is safare 
```
mask_dir = "Train" if filename.startswith("train") else "Test"
return os.path.join(self._root, mask_dir, "Labels", f"{filename}.mat")
```